### PR TITLE
add custom variables support for openx amp-ads

### PR DESCRIPTION
**Pull request by rjvelicaria**
#7937 

add custom variables support for OpenX amp-ads.

### CODE REVIEW

**Comment by rjvelicaria** [comment]
@CLA OpenX Technologies, Inc.

Not my first time contributing. . . this check didn't respond last time. :crossed_fingers:

**Comment by rjvelicaria** [comment]
@CLA OpenX Technologies, Inc.
last CI attempt timed out

File: ads/openx.md [hunk 1]
```diff
@@ -50,19 +57,23 @@ __Required:__
...
 - `data-nc` - Network code '-' sitename
+- `data-auid` - Open X Ad unit id to display
 - `data-dfp-slot` - The Doubleclick slot
 
 __Optional:__
 
-- `json` - Additional json options. Only the "dfp" is currently respected.
+- `json` - Additional json options.
+
+  `customVars` field - please refer to the [documentation](https://docs.openx.com/Content/developers/ad_request_api/custom_variables_in_ad_calls.html).  Also note that OpenX bidder limits these keys by the __whitelisted keys__ set on your publisher settings.
```
Comment (zhouyx) [inline ads/openx.md]: same here

File: ads/openx.md [hunk 2]
```diff
@@ -30,11 +30,18 @@ __Required:__
 - `data-auid` - The ad unit ID to display
 - `data-host` - SSL-enabled OpenX delivery domain
 
+__Optional:__
+
+- `json` - Additional json options.
+
+  `customVars` field - please refer to the [documentation](https://docs.openx.com/Content/developers/ad_request_api/custom_variables_in_ad_calls.html).
```
Comment (zhouyx) [inline ads/openx.md]: I would suggest inline this note to the json param so reader don't think customVars is also an element attribute.

**Comment by rjvelicaria** [comment]
@zhouyx Thanks! I inlined it as suggested.

**Comment by rjvelicaria** [comment]
This build error is not consistent. I get more test passes (around 70% of the time) when I run it on local.

**Comment by rjvelicaria** [comment]
@zhouyx this should be good to go

**Comment by zhouyx** [comment]
@rjvelicaria Thanks for the PR. LGTM. Merged.

**Pull request merged**